psi4/src/psi4/libfock/DirectJK.cc

```diff
@@ -47,7 +47,8 @@
 #include <algorithm>
 #include <limits>
 #include <sstream>
-#include <set>
+#include <map>
```

Comment:
Is there a reason for this include? I did not observe any std::map objects in this file.

And for whatever it's worth, std::unordered_map is more performant than std::map, if there is a std::map object here I am missing.

Reply:
Ah, missed this. Thank you